### DISCUSSION TITLE
Plan for v5 final release?

### DISCUSSION THREAD

**Discussion by HalleyUK** [phalcon/cphalcon#16011] [General]
Hi, 

What does the final release date look like? we have a commitment to upgrade to Phalcon5 from some projects but would like to use the final release rather than a RC release.
Many thanks.

**Comment by niden** [comment]
A RC will be released this week. Currently we are working on documentation, devtools and sample apps. Possibly end of this month depending on how fast we can make it happen :)

As far as documentation, the upgrade guide is nearly done and moving on to each component's documents.

**Reply by wurst-hans** [reply]
Sounds good. I'm not sure, if I'm going to migrate my active projects to v5. Do I remember correctly, that the next major release (v6) will be a pure PHP release? I know, that this will take a long time though, but if v6 is "native" PHP, I might skip v5.

**Comment by HalleyUK** [comment]
Is there an update on this final release timeline please? we committed to go live by the end of this month (Aug).

**Reply by sinbadxiii** [reply]
We started using Phalcon 5 version since alpha, and during the release of beta and RC, we changed only namespaces and a few methods.

The stability of the v5 framework is no worse than version 4. The sooner you upgrade to the new version. all the better :)

**Reply by niden** [reply]
> Is there an update on this final release timeline please? we committed to go live by the end of this month (Aug).

We just released our last RC version today. The next one will be the stable.

Focus: check github issues reported and if need be fix them as fast as possible and documentation.

Possibly end of the month or earlier (I hope) depending on how busy we are (in real life)
